src/Symfony/Component/Intl/Resources/emoji/composer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "repositories": [
+        {
+            "type": "package",
+            "package": {
+                "name": "unicode-org/cldr",
+                "version": "2022.06.29",
+                "source": {
+                    "type": "git",
+                    "url": "https://github.com/unicode-org/cldr",
+                    "reference": "production/2022-06-29-1740z"
+                }
+            }
+        }
+    ],
+    "require": {
+        "php": ">=7.2",
+        "symfony/filesystem": "^6",
+        "symfony/finder": "^6",
+        "unicode-org/cldr": "2022.06.29"
```

Comment:
you might use `*` here, to avoid duplicating the version set in the package repository. It simplifies updates (and there is a single version anyway)

Reply:
Fixed, thanks